### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by Kalissaac**
Fixes a host of "vulnerabilities" and Dependabot PRs

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,33 +12,33 @@
...
     "bulma": "^0.9.3",
     "bulma-divider": "^0.2.0",
-    "bulma-timeline": "^3.0.4",
-    "gridsome": "^0.7.22",
-    "gridsome-plugin-flexsearch": "^1.0.5",
-    "keycloak-js": "^10.0.1",
+    "bulma-timeline": "^3.0.5",
+    "gridsome": "^0.7.23",
+    "gridsome-plugin-flexsearch": "^1.1.0",
+    "keycloak-js": "^11.0.3",
```
Comment (bdsomer) [inline package.json:25]: I see that `keycloack-js` is the only package with a major upgrade (10.x.x -> 11.x.x). Did you test these changes? And why not update to the most recent (17.x.x)? Not sure if [this vulnerability](https://github.com/keycloak/keycloak/security/advisories/GHSA-83x4-9cwr-5487) is relevant but it affects <15.1.1.

**Review by bdsomer — COMMENTED** [review]
Great work. Just want some more info on `keycloak-js`; I'm not familiar with what effects changing that might have on the codebase. Also [this](https://npmgraph.js.org/) might be a useful tool for analyzing what to update. Just upload the `package.json`

**Comment by bdsomer** [comment]
Also, about the "vulnerabilities": https://overreacted.io/npm-audit-broken-by-design/

File: package.json (hunk 1 above)
Comment (Kalissaac) [reply package.json:25]: From my understanding, Keycloak updates all of their packages together: https://www.keycloak.org/docs/latest/release_notes/#keycloak-11-0-0. So I just updated it to the same version that we currently run on our back end: https://github.com/ocf/kubernetes/blob/6fbc98856aefccc078ff4009a8347470c8b9e85d/ocfkube/apps/versions.toml#L42

Comment (Kalissaac) [reply package.json:25]: We could probably upgrade to newer versions without any issues, but I would prefer to upgrade our backend version before that

**Comment by Kalissaac** [comment]
> Also, about the "vulnerabilities": https://overreacted.io/npm-audit-broken-by-design/

Yeah I'm aware, I just wanted to clean up any low-hanging fruit with regards to easy patch fix updates and get rid of the Dependabot PRs really

File: package.json (hunk 1 above)
Comment (bdsomer) [reply package.json:25]: Gotcha, sounds good!

**Review by bdsomer — APPROVED** [review]

**Pull request merged**